packages/frontend/src/components/ControlledAutocomplete/index.tsx

```diff
@@ -213,10 +212,18 @@ function ControlledAutocomplete(
                     key={option.value.toString()}
                     style={{ flexDirection: 'column', alignItems: 'start' }}
                   >
-                    <Typography>{option.label}</Typography>
+                    <Text textStyle="body-1">{option.label}</Text>
+
+                    {option.description && (
+                      <Text textStyle="body-2" color="base.content.medium">
+                        {option.description}
+                      </Text>
+                    )}
 
                     {showOptionValue && (
-                      <Typography variant="caption">{option.value}</Typography>
+                      <Text textStyle="body-2" color="base.content.medium">
+                        {option.value}
+                      </Text>
```

Comment:
nit: right now, we can show both `value` and `description` if we accidentally put both when defining the `field`, can consider showing `description` over the `value` if this happens

Reply:
good call, done